build/bin/user_setup

```diff
@@ -4,6 +4,10 @@ set -euo pipefail
 # ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
 mkdir -p ${HOME}
 chown ${USER_UID}:0 ${HOME}
+chown ${USER_UID}:0 /mattermost-cloud
+# ensure /.aws exists and is accessible by group USER_UID
+mkdir /.aws
```

Comment:
You should use the `-p` flag, otherwise if the directory already exists this will throw an error and the script will fail because of `set -e` line at the top